rotkehlchen/db/accounting_rules.py

```diff
@@ -7,11 +8,22 @@
 from rotkehlchen.db.constants import NO_ACCOUNTING_COUNTERPARTY
 from rotkehlchen.db.filtering import AccountingRulesFilterQuery
 from rotkehlchen.errors.misc import InputError
+from rotkehlchen.logging import RotkehlchenLogsAdapter
 
 if TYPE_CHECKING:
     from rotkehlchen.db.dbhandler import DBHandler
 
 
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+class RuleInformation(NamedTuple):
```

Comment:
Docstring explaining what this is and especially why the identifier is a tuple (this is such a confusing name) don't use identifier as name for a tuple. And what is overwrites etc.